diff -urN gltron-0.61.org/configure.in gltron-0.61/configure.in
--- gltron-0.61.org/configure.in	Mon Apr 30 11:51:06 2001
+++ gltron-0.61/configure.in	Mon Apr 30 11:53:10 2001
@@ -16,8 +16,6 @@
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
 
-CFLAGS=""
-
 AC_ARG_ENABLE(warn,
 [ --enable-warn                Enable compiler warnings [default=on]],
 	enable_warn=$enableval, enable_warn=yes)
@@ -46,7 +44,7 @@
 AC_ARG_ENABLE(optimize,
 [ --enable-optimize=[level]    Enable optimization [default=2]],
 	enable_optmize=$enableval, enable_optimize=2)
-if test "x$enable_optimize" ; then
+if test "x$enable_optimize" != xno -a "x$enable_optimize" ; then
 	CFLAGS="$CFLAGS -O$enable_optimize"
 fi
 
