--- gltron-0.61.org/configure.in	Sun Feb 25 23:14:50 2001
+++ gltron-0.61/configure.in	Thu Dec 26 21:06:14 2002
@@ -16,8 +16,6 @@
 AC_PROG_LN_S
 AC_PROG_MAKE_SET
 
-CFLAGS=""
-
 AC_ARG_ENABLE(warn,
 [ --enable-warn                Enable compiler warnings [default=on]],
 	enable_warn=$enableval, enable_warn=yes)
@@ -46,7 +44,7 @@
 AC_ARG_ENABLE(optimize,
 [ --enable-optimize=[level]    Enable optimization [default=2]],
 	enable_optmize=$enableval, enable_optimize=2)
-if test "x$enable_optimize" ; then
+if test "x$enable_optimize" != xno -a "x$enable_optimize" ; then
 	CFLAGS="$CFLAGS -O$enable_optimize"
 fi
 
@@ -107,12 +105,14 @@
 AC_HEADER_STDC
 AC_CHECK_HEADERS(unistd.h)
 
-AC_CHECK_HEADERS(GL/gl.h,,
- AC_CHECK_HEADERS(/usr/X11R6/include/GL/gl.h,
-	includes_in_x=1,
- AC_CHECK_HEADERS(/usr/local/include/GL/gl.h,
-	includes_in_local=1,
-AC_MSG_ERROR(cant find GL headers))))
+CPPFLAGS="$CPPFLAGS -I/usr/X11R6/include"
+
+AC_CHECK_HEADERS(GL/gl.h,[],
+ [AC_CHECK_HEADERS(/usr/X11R6/include/GL/gl.h,
+	[includes_in_x=1],
+ [AC_CHECK_HEADERS(/usr/local/include/GL/gl.h,
+	[includes_in_local=1],
+[AC_MSG_ERROR(cant find GL headers)])])])
 
 if test "$includes_in_x" = 1 ; then
 	INCS="$INCS -I/usr/X11R6/include"
@@ -122,6 +122,8 @@
 	INCS="$INCS -I/usr/local/include"
 fi
 
+INCS="$INCS $CPPFLAGS"
+
 AC_SUBST(INCS)
 
 dnl Checks for typedefs, structures, and compiler characteristics.
